server/src/test/java/org/elasticsearch/action/support/IndicesOptionsTests.java

```diff
@@ -134,4 +134,21 @@ public void testFromParameters() {
         assertEquals(defaultOptions.forbidClosedIndices(), updatedOptions.forbidClosedIndices());
         assertEquals(defaultOptions.ignoreAliases(), updatedOptions.ignoreAliases());
     }
+
+    public void testSimpleByteBWC() {
+        // Corresponds to byte 24 prior to rewriting IndicesOptions not to be byte-based
+        IndicesOptions indicesOptions = IndicesOptions.fromOptions(false, false, false, true, false, false, false);
+
+        logger.info("--> 1 {}", indicesOptions.toString());
+        IndicesOptions indicesOptions2 = IndicesOptions.fromByte((byte)24);
```

Comment:
maybe we should expand testing a bit, just to make sure? the byte representation is quite convenient as it doesn't require the usual base64 based testing for these bw comp scenario. I think that we should test more of the previous values, if not all.

Reply:
Sounds good, I've enhanced this test